EncoreTickets.SDK/Api/Results/Exceptions/ApiException.cs

```diff
@@ -93,77 +88,75 @@ public ApiException(string message)
             Response = response;
         }
 
-        protected string GetMessage()
+        /// <summary>
+        /// Returns easily read errors that are the cause of the exception.
+        /// </summary>
+        /// <returns>Collection of strings</returns>
+        protected List<string> GetErrors()
         {
-            if (string.IsNullOrEmpty(predefinedMessage))
-            {
-                return Errors.Any() ? string.Join("; ", Errors) : DefaultMessage;
-            }
-
-            return predefinedMessage;
+            var errors = GetErrorsAsString(Response, ContextInResponse);
+            return FilterErrorsAsStrings(errors);
         }
 
         /// <summary>
-        /// Returns easily read errors that are the cause of the exception.
+        /// Returns a string contained easily read info about errors that are the cause of the exception.
         /// </summary>
-        /// <returns></returns>
-        protected List<string> GetErrors()
+        /// <returns>Message about errors</returns>
+        protected string GetMessage()
         {
-            if (ContextInResponse?.Errors == null)
+            if (predefinedMessage != null)
             {
-                return new List<string>
-                {
-                    string.IsNullOrEmpty(Response.StatusDescription)
-                        ? Response.ErrorMessage
-                        : Response.StatusDescription
-                };
+                return predefinedMessage;
             }
 
-            var contextErrors = ContextInResponse.Errors.Select(ConvertErrorToString)
-                .Where(x => !string.IsNullOrEmpty(x));
-            return contextErrors.ToList();
+            return Errors != null && Errors.Any()
+                ? string.Join("; ", Errors)
+                : DefaultMessage;
         }
 
-        private string ConvertErrorToString(Error error)
+        protected List<string> FilterErrorsAsStrings(IEnumerable<string> errors)
```

Comment:
This method looks like it's not specific to errors but rather belongs to general IEnumerable extensions